src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -105,6 +111,23 @@ PluginResponse handlePluginsRequest(PluginRequest pluginRequest) {
         return pluginResponse;
     }
 
+    /**
+     * Handles a request from OpenSearch and invokes the extension point API corresponding with the request type
+     *
+     * @param request  The request to handle.
+     * @return A response to OpenSearch for the corresponding API
+     */
+    TransportResponse handleOpenSearchRequest(OpenSearchRequest request) {
```

Comment:
Need test for this function is TestExtensionRunner

Reply:
just added the test